index.html

```diff
@@ -194,6 +194,10 @@ <h2>Terminology</h2>
 An <dfn>integer</dfn> is a <a>Number</a> that is unchanged
 under the <a>ToInteger</a> operation.
 
+<p>A <dfn>millisecond timestamp</dfn> is an <a>Integer</a> that
```

Comment:
`Date.prototype.toJSON` delegates to [Date.prototype.toISOString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) I suggest we lean into that, rather than epoch millis